### ISSUE TITLE
Create a mediarestore command to reflect mediabackup (From TODO.md)

### ISSUE THREAD

**Issue by ZuluPro** [django-dbbackup/django-dbbackup#84]
https://github.com/django-dbbackup/django-dbbackup/blob/5bb44b8a7ca4ae4bb973a88b44cf938431a88795/TODO.md

**Comment by benjaoming** [comment]
:+1:

**Comment by regit** [comment]
Is someone working on this ?

**Comment by mjs7231** [comment]
I don't think anyone picked it up as a project yet.

**Comment by ZuluPro** [comment]
It is not in my priorities because outside of FileSystemStorage mediarestore is useless. Backup a cloud ...

But I know it could be very useful for development files for example. So if you really need it, do not hesitate to tell us.

**Comment by regit** [comment]
Thanks for your answer. In the case of my application https://github.com/StamusNetworks/scirius, I've got django objects and I've got a a list of directories with data provided by users. This data is essential for the application and I can not just have the Django objects. Si if I want to offer a real backup/restore to the user I need to be able to backup and restore this directories and the db at the same time.
